ethereum/consensus-core/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version.workspace = true
 edition = "2021"
 
 [dependencies]
-alloy = { version = "0.9.1", features = [
+alloy = { workspace = true, features = [
```

Comment:
By importing this as a workspace import it means we inherit all of the features that are flagged in the top level workspace. I worry that this will break some usecases like various zk provers (although I guess sp1 doesn't run into an issue here). Still probably worthwhile to leave it as its own import for now just to ensure that this critical piece only flags the minimal number of features.